spec/integration/data_stream/manifest.spec.yml

```diff
@@ -102,6 +104,40 @@ spec:
           default:
             description: Default value(s) for variable
             $ref: "#/definitions/input_variable_value"
+        if:
+          properties:
+            type:
+              const: select
+        then:
+          required:
+            - options
+          properties:
+            options:
+              description: List of options for select type
+              type: array
+              items:
+                type: object
+                additionalProperties: false
+                properties:
+                  value:
+                    type: string
+                    examples:
+                    - node
+                    - cluster
+                  text:
+                    type: string
+                    examples:
+                    - node
+                    - cluster
+                required:
+                - value
+                - text
+              min_items: 1
+              default: []
+        else:
+          not:
+            required:
+              - options
         required:
```

Comment:
As you prefer. Historically the main reasons we have had to force the use of a spec version are:
* Change is potentially breaking, so we don't want it to apply to packages using old versions of the spec (this doesn't seem to be the case here).
* Encouraging developers to use more recent versions of the spec.

In the future we may be more strict, and always introduce new features in new versions. If we want to follow this approach already here, this feature should be introduced in 2.6.0. For that we would need to use `before: 2.6.0`, and add a `2.6.0-next` block in the changelog.

Reply:
Thanks @jsoriano , that sounds good to me. @mrodm WDYT?

Reply:
Agreed! :+1:

Reply:
Mmh, there seems to be a catch 22: that makes the tests fail as version is not 2.6.0 yet :) Does this need to go into the next PR?

Reply:
What is the error ? 
Just in case, if this is added just from spec version >= 2.6.0, `format_spec` field in the manifests of the test packages should be updated to at least that version.

Reply:
Thanks, it works now!